package.json

```diff
@@ -2,12 +2,16 @@
   "name": "@netlify/framework-info",
   "version": "8.0.2",
   "description": "Framework detection utility",
-  "main": "./src/main.js",
-  "browser": "./dist/index.js",
+  "type": "module",
+  "main": "./dist/index.cjs",
```

Comment:
The `main` field is needed for Webpack and Jest to work correctly when used by browser consumers like our front-end app.

Reply:
shouldn't we use the `browsers` field instead of the main field if it is used by browsers?

https://docs.npmjs.com/cli/v8/configuring-npm/package-json#browser

Reply:
I have tried several combinations of `main`, `browsers`, or both, using both `./src/main.js` and/or `./dist/index.cjs` and this ended up being the combination which worked with our frontend application.

The main problem is that the frontend application is using several tools which are not quite completely ready for pure ES modules and have their own interpretation of how those fields should be used: Webpack, Jest, `tsc`.